### PR TITLE
NSH: fix layer binding for VXLAN GPE

### PR DESCRIPTION
**Pull request by raslandarawsheh**
Use correct field name to bind VXLAN GPE

**Checklist:**

-   [x] If you are new to Scapy: I have checked [CONTRIBUTING.md](https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md) (esp. section submitting-pull-requests)
-   [x] I squashed commits belonging together
-   [x] I added unit tests or explained why they are not relevant
-   [x] I executed the regression tests (using `cd test && ./run_tests` or `tox`)
-   [ ] If the PR is still not finished, please create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)

fixes #xxx

### CODE REVIEW

File: test/contrib/nsh.uts [hunk 1]
```diff
@@ -14,3 +14,6 @@ raw(Ether(src="00:00:00:00:00:01", dst="00:00:00:00:00:02")/IP(src="1.1.1.1", ds
 
 = 0 length variable length context header NSH
 raw(NSH(mdtype=2, spi=0xF0F0F0, si=0xFF))  == b'\x0f\xc2\x02\x03\xf0\xf0\xf0\xff'
+
+= Build a NSH over VXLAN packet and verify bindings
+raw(Ether(dst='0c:42:a1:5f:fb:e0',src='b8:59:9f:cd:de:3e')/IPv6()/UDP(dport=8472)/VXLAN(vni=0x1234, NextProtocol=4)/NSH()/NSH()/Ether()/IP(src="10.200.100.10", dst="2.2.2.3")/TCP()) == b'\x0cB\xa1_\xfb\xe0\xb8Y\x9f\xcd\xde>\x86\xdd`\x00\x00\x00\x00v\x11@\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x12\xb5!\x18\x00v\xf2\x96\x0c\x00\x00\x04\x00\x124\x00\x0f\xc6\x01\x04\x00\x00\x00\xff\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x0f\xc6\x01\x03\x00\x00\x00\xff\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff\xff\xff\xff\xff0\xe1qU=\xb0\x08\x00E\x00\x00(\x00\x01\x00\x00@\x06\x07\xf9\n\xc8d\n\x02\x02\x02\x03\x00\x14\x00P\x00\x00\x00\x00\x00\x00\x00\x00P\x02 \x00\x1c\xa8\x00\x00'
```
Comment (gpotter2) [inline test/contrib/nsh.uts:19]: You should hardcode all addresses (Ethernet and IPv6) for tests to pass.

**Review by gpotter2 — COMMENTED** [review]
Thanks for the PR ! tests are failing

File: test/contrib/nsh.uts (hunk 1 above)
Comment (raslandarawsheh) [reply test/contrib/nsh.uts:19]: sure will handle

**Review by guedou — APPROVED** [review]
Thanks

**Pull request merged**